pwa/src/App.tsx

```diff
@@ -35,14 +36,24 @@ const App: React.FunctionComponent = () => {
                         </Wrapper>
                     </Route>
                     <Route path="/diagnosis">
-                    <Wrapper
-                        navTitle="Diagnosis"
-                        leftArrowText="Patient Info"
-                        leftArrowRoute="/patientinfo"
-                    >
-                        <DiagnosisForm />
-                    </Wrapper>
-                </Route>
+                        <Wrapper
+                            navTitle="Diagnosis"
+                            leftArrowText="Patient Info"
+                            leftArrowRoute="/patientinfo"
+                        >
+                            <DiagnosisForm />
+                        </Wrapper>
+                    </Route>
+                    <Route path="/tests">
+                        <Wrapper
+                            navTitle="Tests"
+                            // TODO: uncomment when Treatment component is completed
+                            // leftArrowText="Treatment"
+                            // leftArrowRoute="/treatment"
```

Comment:
This should be done as part of either #104 or #103, depending on which is merged first.